.circleci/config.yml

```diff
@@ -18,6 +18,10 @@ jdk_8: &jdk_8
 jdk_11: &jdk_11
   JDK_VERSION: 11
 
+unit_test_env: &unit_test_env
```

Comment:
needed to unit tests, because code now is using `sys.env ` there.
I think that would be nice to make use of layers there, and instead of getting sys.env directly, take it from ZIO Env.
So will be easier to unit test it.

Reply:
👍